src/mlpack/methods/dbscan/dbscan_impl.hpp

```diff
@@ -181,7 +177,8 @@ void DBSCAN<RangeSearchType, PointSelectionPolicy>::PointwiseCluster(
     visited[index] = true;
 
     // Do the range search for only this point.
-    rangeSearch.Search(data.col(index), Range(0.0, epsilon), neighbors,
+    rangeSearch.Search(data.col(index), RangeType<ElemType>(ElemType(0.0), epsilon),
+        neighbors,
         distances);
```

Comment:
```suggestion
        neighbors, distances);
```
(Just a small cleanup.)